webnode/src/components/consent-overlay/middle-group/index.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from "react";
```

Comment:
This is the proper way of defining imports but there is a new way, which is `import React form "react"`. 
I noticed that you updated this, and it is not wrong but you dont have to change this.  Check out     "allowSyntheticDefaultImports" on the TS compiler options. I vote we put it back to the old way for simpler imports so you would have to turn this on.

Reply:
its right by definition, precisely due to typology